### ISSUE TITLE
PathNet: Evolution Channels Gradient Descent in Super Neural Networks

### ISSUE THREAD

**Issue by TMats** [TMats/survey#70]
https://arxiv.org/abs/1701.08734
